### PR TITLE
Add Vicuna example folder.

### PR DESCRIPTION
**Pull request by concretevitamin**
Tested (run the relevant ones):

- [ ] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `bash tests/backward_comaptibility_tests.sh`

### CODE REVIEW

File: examples/vicuna-llm/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Vicuna: An LLM Chatbot Impressing GPT-4 with 90% ChatGPT Quality
+
+<img src="https://vicuna.lmsys.org/favicon.jpeg" width=256 alt="Vicuna LLM"/>
```
Comment (Michaelvll) [inline examples/vicuna-llm/README.md:3]: Should we use percentage width instead to make it looks better on different screens?

File: examples/vicuna-llm/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# Train a 7B model instead of the default 13B
+sky spot launch -n vicuna-7b scripts/train-vicuna.yaml --env WANDB_API_KEY --env MODEL_SIZE=7
+
+# Use *unmanaged* spot instances (i.e., preemptions won't get auto-recovered).
+# Unmanaged spot saves the cost of a small controller VM.  We recommend using managed spot as above.
+sky launch -n vicunab scripts/train-vicuna.yaml --env WANDB_API_KEY
+```
+Currently, such instances (`--gpus A100-80GB:8 --use-spot`) are only available on GCP.
+
+**To use on-demand `A100-80GB:8` instances**, which are currently available on both Azure and GCP:
```
Comment (Michaelvll) [inline examples/vicuna-llm/README.md:40]: I think lambda is also supported.

**Review by Michaelvll — APPROVED** [review]
Thanks for adding this @concretevitamin ! LGTM.

File: examples/vicuna-llm/README.md (hunk 1 above)
Comment (concretevitamin) [reply examples/vicuna-llm/README.md]: Done, set to 25%.

File: examples/vicuna-llm/README.md (hunk 2 above)
Comment (concretevitamin) [reply examples/vicuna-llm/README.md]: Good catch, done.

**Pull request merged**